polkadot-parachain/src/chain_spec/asset_hubs.rs

```diff
@@ -19,61 +19,66 @@ use crate::chain_spec::{
+pub type AssetHubPolkadotChainSpec =
+	sc_service::GenericChainSpec<asset_hub_polkadot_runtime::GenesisConfig, Extensions>;
+pub type AssetHubKusamaChainSpec =
+	sc_service::GenericChainSpec<asset_hub_kusama_runtime::GenesisConfig, Extensions>;
+pub type AssetHubWestendChainSpec =
+	sc_service::GenericChainSpec<asset_hub_westend_runtime::GenesisConfig, Extensions>;
 
-const STATEMINT_ED: StatemintBalance = statemint_runtime::constants::currency::EXISTENTIAL_DEPOSIT;
-const STATEMINE_ED: StatemintBalance = statemine_runtime::constants::currency::EXISTENTIAL_DEPOSIT;
-const WESTMINT_ED: StatemintBalance = westmint_runtime::constants::currency::EXISTENTIAL_DEPOSIT;
+const ASSET_HUB_POLKADOT_ED: AssetHubBalance =
+	asset_hub_polkadot_runtime::constants::currency::EXISTENTIAL_DEPOSIT;
+const ASSET_HUB_KUSAMA_ED: AssetHubBalance =
+	asset_hub_kusama_runtime::constants::currency::EXISTENTIAL_DEPOSIT;
+const ASSET_HUB_WESTEND_ED: AssetHubBalance =
+	asset_hub_westend_runtime::constants::currency::EXISTENTIAL_DEPOSIT;
 
 /// Generate the session keys from individual elements.
 ///
 /// The input must be a tuple of individual keys (a single arg for now since we have just one key).
-pub fn statemint_session_keys(keys: StatemintAuraId) -> statemint_runtime::SessionKeys {
-	statemint_runtime::SessionKeys { aura: keys }
+pub fn asset_hub_polkadot_session_keys(
+	keys: AssetHubPolkadotAuraId,
+) -> asset_hub_polkadot_runtime::SessionKeys {
+	asset_hub_polkadot_runtime::SessionKeys { aura: keys }
 }
 
 /// Generate the session keys from individual elements.
 ///
 /// The input must be a tuple of individual keys (a single arg for now since we have just one key).
-pub fn statemine_session_keys(keys: AuraId) -> statemine_runtime::SessionKeys {
-	statemine_runtime::SessionKeys { aura: keys }
+pub fn asset_hub_kusama_session_keys(keys: AuraId) -> asset_hub_kusama_runtime::SessionKeys {
+	asset_hub_kusama_runtime::SessionKeys { aura: keys }
 }
 
 /// Generate the session keys from individual elements.
 ///
 /// The input must be a tuple of individual keys (a single arg for now since we have just one key).
-pub fn westmint_session_keys(keys: AuraId) -> westmint_runtime::SessionKeys {
-	westmint_runtime::SessionKeys { aura: keys }
+pub fn asset_hub_westend_session_keys(keys: AuraId) -> asset_hub_westend_runtime::SessionKeys {
+	asset_hub_westend_runtime::SessionKeys { aura: keys }
 }
 
-pub fn statemint_development_config() -> StatemintChainSpec {
+pub fn asset_hub_polkadot_development_config() -> AssetHubPolkadotChainSpec {
 	let mut properties = sc_chain_spec::Properties::new();
 	properties.insert("ss58Format".into(), 0.into());
 	properties.insert("tokenSymbol".into(), "DOT".into());
 	properties.insert("tokenDecimals".into(), 10.into());
 
-	StatemintChainSpec::from_genesis(
+	AssetHubPolkadotChainSpec::from_genesis(
 		// Name
-		"Statemint Development",
+		"Polkadot Asset Hub Development",
```

Comment:
No, the natural language name (like what to display to users on a UI) would be the "Polkadot Asset Hub".

Reply:
if it's UI then displaying Asset Hub Polkadot would be nicer - if you have a few of these and they are truncated then in it was Polkadot first you'd just see a list of "Polkadot", "Polkadot" rather than "Asset H", "Bridge H"

Reply:
OK, maybe I'm wrong. Good point on truncation. We don't say "Relay Chain Polkadot" though, so don't know why you'd reverse it for parachains.

Within the codebase, I still think `asset-hub-polkadot` / `AssetHubPolkadot` makes more sense, since you want imports/filestructure grouped by functionality, not relay.